python/tools/build_tx.py

```diff
@@ -30,10 +30,10 @@
 SESSION.headers.update({"User-Agent": "trezorlib"})
 
 BITCOIN_CORE_INPUT_TYPES = {
-    "scripthash": messages.InputScriptType.SPENDADDRESS,
-    "pubkeyhash": messages.InputScriptType.SPENDP2SHWITNESS,
+    "pubkeyhash": messages.InputScriptType.SPENDADDRESS,
+    "scripthash": messages.InputScriptType.SPENDP2SHWITNESS,
```

Comment:
it's not just for echoing, it's auto-detection of the script type that should be set on `TxInput`.

it can't be `EXTERNAL` or `MULTISIG` here because those aren't supported by build_tx (the input would be badly constructed anyway so no point in worrying about the right script_type)

(fwiw any of the types could be `EXTERNAL` if we wanted to support that, right?)

Reply:
added a short comment in 28e70a1

Reply:
:+1